test/rng_testsuite/extra_methods_tests/distributions_methods.pass.cpp

```diff
@@ -0,0 +1,269 @@
+// -*- C++ -*-
+//===-- engines_methods.hpp ----------------------             -------------===//
+//
+// Copyright (C) Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// This file incorporates work covered by the following copyright and permission
+// notice:
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+//
+//===----------------------------------------------------------------------===//
+//
+// Abstract:
+//
+// Testing of different distributions' methods
+
+#include "support/utils.h"
+#include <iostream>

```

Comment:
Could you please clarify why we need `iostream` for non-SYCL cases? I didn't see it's usage in non-SYCL code

Reply:
transferred under the define